{{cookiecutter.repostory_name}}/.github/workflows/ci.yml

```diff
@@ -39,14 +39,16 @@ jobs:
         with:
           python-version: {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
           cache: "pip"
-      - name: Prepare environment
-        run: ./setup-dev.sh
       - name: Install dependencies
         run: python -m pip install --upgrade nox 'pdm>=2.12,<3'
+      - name: Prepare environment
+        run: ./setup-dev.sh
       - name: Run dockerized services
         run: docker compose up -d --wait
       - name: Run migrations
         run: cd app/src && pdm run python manage.py wait_for_database --timeout 120 && pdm run python manage.py migrate
+      - name: Run static file collection
+        run: cd app/src && pdm run python manage.py collectstatic
```

Comment:
why is this needed?

Reply:
Some tests fail without this step with this error:
`ValueError: Missing staticfiles manifest entry for 'img/favicon.png'`

https://github.com/reef-technologies/reef.pl/actions/runs/8616637087/job/23614679098

Reply:
```python
=========================== short test summary info ============================
FAILED reefpl/github/tests.py::GitHubTests::test_account_name_null - ValueError: Missing staticfiles manifest entry for 'img/favicon.png'
FAILED reefpl/github/tests.py::GitHubTests::test_authentication_error - ValueError: Missing staticfiles manifest entry for 'img/favicon.png'
FAILED reefpl/github/tests.py::GitHubTests::test_login - ValueError: Missing staticfiles manifest entry for 'img/favicon.png'
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 3 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
```

But I have removed these tests anyway, since they are just copy-pasted from `django-allauth`, and fail even in the old `reef.pl` (before `cruft update`). 

So now we don't need `collectstatic` anymore, so we can remove it then? I'm not sure.

Reply:
as for manifest thing, I think it occurs there since 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage' is used there, so unless we want to introduce it into cookiecutter as well then I don't see reason for a fix for it to be introduced in cookiecutter and not there alone.

We can discuss private repo changes in that repo PR alone.
Here lets make sure we are addressing only problems that are common to all projects using cookiecutter.

Reply:
Sorry about that, I forgot this is a public repo.

I have reverted the `collectstatic` step, indeed it is not required for all projects.